piccolo/apps/asgi/commands/templates/app/home/_xpresso_endpoints.py.jinja

```diff
@@ -0,0 +1,21 @@
+import os
+
+import jinja2
+from starlette.endpoints import HTTPEndpoint
+from starlette.responses import HTMLResponse
```

Comment:
```suggestion
from xpresso.responses import HTMLResponse
```

I did the same re-export as most other Starlette based frameworks do.
I figure more than for convenience I'd do it so that if at some point I need to subclass `Request` or something I can maybe do it without breaking people's code.